# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/hotfix-cpu-test.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

 0:01.58 DEBUG: CPU = 1

as gcc defines a sparc built-in, just undef it, sigh

--- Arctic-Fox-46.0/build/moz.configure/toolchain.configure.vanilla	2025-07-22 23:02:17.227029547 +0200
+++ Arctic-Fox-46.0/build/moz.configure/toolchain.configure	2025-07-22 23:21:45.847702564 +0200
@@ -259,6 +259,7 @@
         for n, (value, condition) in enumerate(preprocessor_checks.iteritems()):
             check += dedent('''\
                 #%(if)s %(condition)s
+                #undef %(value)s
                 %%%(name)s %(value)s
             ''' % {
                 'if': 'elif' if n else 'if',
